project/spreadsheet.py

```diff
@@ -257,6 +256,17 @@ def initialize(email, configMap, arg):
 
             SPREADSHEET_ID = sheetCreated['spreadsheetId']
 
+            if 'spreadsheet_database' in configMap:
+                folderID = configMap['spreadsheet_database']['folderID']
+            else:
+                print("Spreadsheet_database section is missing in your config file")
+                return None
+
```

Comment:
Setup the service first and then pass the service object into the moveFileToFolder function